modules/ROOT/pages/dw-operators.adoc

```diff
@@ -491,7 +505,7 @@ payload update {
 
 ==== Index Selector
 
-This matching selector allows to match any array by its index. +
+This matching selector allows to match any array by its index.
```

Comment:
this wording doesn't seem correct, can you verify. allows what to match?

Reply:
fixed here and in another sentence like it